mne/viz/topomap.py

```diff
@@ -171,24 +178,38 @@ def _prepare_topomap_plot(inst, ch_type, sphere=None):
             # change names so that vectorview combined grads appear as MEG014x
             # instead of MEG0142 or MEG0143 which are the 2 planar grads.
             ch_names = [ch_names[k][:-1] + "x" for k in range(0, len(ch_names), 2)]
-        else:
-            assert ch_type in _fnirs_types
-            # Modify the nirs channel names to indicate they are to be merged
+        elif modality == "fnirs":
+            # Modify the channel names to indicate they are to be merged
             # New names will have the form  S1_D1xS2_D2
             # More than two channels can overlap and be merged
             for set_ in overlapping_channels:
                 idx = ch_names.index(set_[0][:-4])
                 new_name = "x".join(s[:-4] for s in set_)
                 ch_names[idx] = new_name
+        elif modality == "opm":
+            # Modify the channel names to indicate they are to be merged
+            # New names will have the form  S1xS2
+            for set_ in overlapping_channels:
+                idx = ch_names.index(set_[0])
+                new_name = ".".join(s for s in set_)
```

Comment:
If we're going to join based on a `.` and then check it later, better raise an error here if there is already a `.` in the name. But maybe I'm missing a check somewhere else. And maybe a moot point if you take my idea above